src/System.Runtime/src/System/HashCode.cs

```diff
@@ -0,0 +1,406 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Please follow the rules from the "Copying Files from Other Projects" section of https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/contributing.md. In particular, I think that means including the xxHash license here and adding it to the 3rd party notices file. @richlander, please confirm we don't need to do anything else to make this file follow the rules.